src/backendbot/utils.py

```
@@ -112,25 +128,41 @@ def save_memory(memory):
+    if AsyncSessionLocal is None:
+        log_event(f"DB no disponible - Optimización: {freed_ram_mb:.2f}MB liberados")
+        return
+
+    try:
+        async with AsyncSessionLocal() as session:
+            new_entry = OptimizationEvent(
+                timestamp=time.time(), freed_ram_mb=freed_ram_mb
+            )
+            session.add(new_entry)
+            await session.commit()
+    except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)